CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## vNEXT
+- Harmonize SRLC wording. (#31)
```

Comment:
Could be in Cleans todos ?

Reply:
Todo section is too far in the bottom imo!
Maybe when all is completed we can sort/group bullets in changelog (?)